### DISCUSSION TITLE
Pimcore System Requirements in future with Pimcore Studio

### DISCUSSION THREAD

**Discussion by ascheider** [pimcore/pimcore#17936] [Q&A / Support] [answered]
Hey folks,

tried to setup current studio preview and have some questions.
With the studio ui pimcore needs generic index bundle https://docs.pimcore.com/platform/Generic_Data_Index/Installation/ which requires ES or OpenSearch.
It also requires studio backend bundle https://github.com/pimcore/studio-backend-bundle which requires mercury.
So pimcore installation will in future always need at least ES/OpenSearch and Docker? Or do i miss something?
Thanks!

**Comment by dpfaffenbauer** [comment, marked as answer]
nothing needs docker, but Studio needs ES/OS and Mercure, yes.

**Reply by ascheider** [reply]
Hi Dominik,

thanks for the reply.
you are right, the docker ist not necessary, but i think it's easier to setup mercury with it: https://mercure.rocks/docs/hub/install

**Reply by dpfaffenbauer** [reply]
definitely ;)

**Reply by Chauhan-Mukesh** [reply]
Is ES/OS compulsory to have or good to have?

**Reply by dpfaffenbauer** [reply]
its a must have
